<launch>
		<node pkg="visual_servoing" name="tag_detector" type="tag_detection_server" output="screen"/>
		<node pkg="visual_servoing" name="tracker" type="tracker_server" output="screen"/>
		<node pkg="visual_servoing" name="controller" type="landing_control" output="screen"/>
</launch>
